doc_source/programming-key-policies.md

```diff
@@ -96,6 +96,24 @@ $result = $KmsClient->listKeyPolicies([
 ]);
 ```
 
+------
+#### [ Node.js ]
+
+For details, see the [ListKeyPolicies property](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/KMS.html#listKeyPolicies-property) in the *AWS SDK for Node.js*\.
+
+```js
+// List key policies
+//
+// Replace the following fictitious CMK ARN with a valid CMK ID or ARN
+
+const KeyId = 'arn:aws:kms:us-west-2:111122223333:key/1234abcd-12ab-34cd-56ef-1234567890ab';
+const Limit = 10;
```

Comment:
@juneb PHP has a limit, but Ruby, Python et. al. don't.  I would default to having people put in a limit.  Does this mean we should update the other languages?

Reply:
I agree that Limit is a best practice, but given that this API returns only *default*, it's not needed here. Actually, I bet they added this example to show people that when they use this API and always get back *default* and nothing else, it's intentional. They haven't made a mistake.

Let me check with the writer for the PHP SDK. Unless it's critical to PHP for some reason, we should remove it and omit it from the Node.js.

Reply:
Good catch!

No, I don't think so. Limit is a best practice for List APIs, but this one always returns only *default*. I bet they added this example to reassure folks that when they always get *default*, they haven't made a mistake. 

I'll check with the PHP writer. If this isn't critical to the PHP code for some reason specific to PHP, I'll remove it from the PHP example. We shouldn't include it in the Node.js example.

Reply:
I removed the limit from here but not PHP's. Going to do a separate PR for that so we don't get confused. Also, note that I removed the limit only on this specific call for node.js. Hopefully, that's ok, if you want me to remove it from other calls too let me know no problem 👍